### DISCUSSION TITLE
Homebridge Inkbird

### DISCUSSION THREAD

**Discussion by richierockskool** [Zacknetic/HomebridgeMagicHome-DynamicPlatform#139] [Show and tell]
Zack, been working on this one for a while now and totally stuck, trying to use some ideas from MagicHome, but wrote it in JS, please take a look, and help would be great!

https://github.com/richierockskool/homebridge-inkbird-wifi-gateway

**Comment by Zacknetic** [comment]
Hey absolutely. Let’s take a look. Can you message me in the magichome discord server and we’ll setup a collaboration session?

I would like to set expectations that I’ve worked with Noble in the past and have faced roadblocks but we can try.

**Comment by richierockskool** [comment]
Okay I sent you a message, Noble might have to go, all the information in on the Inkbird Wifi Gateway, I think all that has to done get the ip address for the device (similar to your plugin) bring all that info into HK.
